www_src/components/color-spectrum/color-spectrum.jsx

```diff
@@ -1,20 +1,116 @@
 var React = require('react');
+var Slider = require('../../components/range/range.jsx');
+var Color = require('color');
 
-// Doesn't do much yet, but renders the hue and saturation spectrum
+var HSB = React.createClass({
+  getBackgroundColor: function () {
+    var color = this.props.color.clone();
+    color.hsv(color.hue(), 100, 100);
```

Comment:
because of the way the colour spectrum is supposed to look, this is  `hsv/b`, not `hsl`